libs/zzz/ui/src/Wengine/WengineCard.tsx

```diff
@@ -44,18 +47,31 @@ const wenginePhaseIconsMap: Record<PhaseKey, WenginePhaseKey> = {
 
 export function WengineCard({
   wengineId,
+  onClick,
   onEdit,
   setLocation,
   extraButtons,
 }: {
   wengineId: string
-  onClick?: () => void
+  onClick?: (wengineId: string) => void
```

Comment:
nit: The onclick param here is a bit redundant, because from the parent perspective, it already know what `wengineId` is(its passed as another prop), so passing it in wouldn't make sense.